platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/config/GossipConfig.java

```diff
@@ -37,8 +37,8 @@ public record GossipConfig(
      * @param nodeId the node ID
```

Comment:
The comments on this method need to be updated to reflect the field name change.